### DISCUSSION TITLE
Concatenate videos with different frame number

### DISCUSSION THREAD

**Discussion by aledifil** [miniscope/minian#151] [Q&A]
Hi all! I'm having some issues with the way Minian is chunking the videos.

I'm loading videos from 2 different folders, rename them to be consecutive, and run Minian on them all.

My issue is that during the save result part, where I save Y_fm_chk, I get the following error:

ValueError: Zarr requires uniform chunk sizes except for final chunk. Variable named 'Y_fm_chk' has incompatible dask chunks: ((1000, 1000, 1000, 1000, 1000, 1000, 589, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 1000, 890), (411,), (421,)). Consider rechunking using `chunk()`.

I tried chunking Y again but did not work. 
Is there anything else I can do, without making another piece of code to change the frame format?
Thanks!

**Comment by phildong** [comment]
Hey hmm this could be tricky. A few questions:

1. do your videos have same spatial dimensions (height and width)?
2. What happened when you do `Y_fm_chk.chunk({'frame': 1000})`? I assume this should "work" but you might run into memory issues.
